src/mind/mind-elementalist.cpp

```diff
@@ -800,7 +783,7 @@ static bool get_element_power(PlayerType *player_ptr, SPELL_IDX *sn, bool only_b
 
                     PERCENTAGE chance = decide_element_chance(player_ptr, spell);
                     int mana_cost = decide_element_mana_cost(player_ptr, spell);
-                    get_element_effect_info(player_ptr, i, comment);
+                    auto comment = get_element_effect_info(player_ptr, i);
```

Comment:
Variables other than primitive types which don't change are desirable to be declared with const.
Would you please add it?
(This is the same for other PRs.)

Reply:
The latest forced push adds const to the declaration at mind/mind-elementalist.cpp:786.